hugo/content/3-queries/introduction.md

```diff
@@ -0,0 +1,169 @@
++++
+next = "/3-queries/operators"
+prev = "/3-queries"
```

Comment:
I can't add a comment in the PR in the right place, but you need to set the `next` of `/3-queries` to `/3-queries/introduction` instead of what it is now.